src/scheduler.js

```diff
@@ -90,8 +88,11 @@ class Schedule {
   // Run job once.
   runOnce() {
     const now = Date.now();
-    if (now >= this.startTime && (!this.endTime || now < this.endTime))
+    if (now >= this.startTime && (!this.endTime || now < this.endTime)) {
+      if (this.every)
+        this.startTime = this._getNextStartTime();
```

Comment:
Apparently, this works as long as the clock always moves forward in time. But as long as some test decides to run at an earlier time, its scheduled jobs will never run. So the solution was to a) add a method called `resetSchedule` method that sets the start time on all the schedules, and b) always call it from `runOnce`, whether it executed a job or not.

https://github.com/assaf/ironium/commit/1221139e3ba156bdfdf910f8f24da2c8cff2fe1f